CODEOWNERS

```diff
@@ -1,15 +1,16 @@
 # default to platform-core
 * @puppetlabs/platform-core
+* @puppetlabs/puppetserver-maintainers
```

Comment:
Isn't there some annoying behavior where @puppetlabs/puppetserver-maintainers will override platform-core?
does it need to be on the same line?
```
* @puppetlabs/platform-core @puppetlabs/puppetserver-maintainers
```

Reply:
oh maybe

Reply:
I forgot about that

Reply:
will fix